### PR TITLE
Isaicp 4471: Unify the filters on the members page and expose them

### PR DESCRIPTION
**Pull request by idimopoulos**
The filters are unified for both moderators and unprivileged users and in addition, for the non privileged users, the filters are now exposed. Theme is needed.

### CODE REVIEW

File: web/profiles/joinup/config/install/views.view.members_overview.yml [hunk 1]
```diff
@@ -122,83 +131,245 @@ display:
...
             default_group_multiple: {  }
-            group_items: {  }
-          entity_type: user
-          entity_field: status
-          plugin_id: boolean
+            group_items:
+              1: {  }
+              2: {  }
+              3: {  }
+          fields:
```
Comment (claudiu-cristea) [inline web/profiles/joinup/config/install/views.view.members_overview.yml]: I find useful to add here also the E-mail. Of course it should be added first as hidden field.

File: web/profiles/joinup/config/install/views.view.members_overview.yml [hunk 2]
```diff
@@ -122,83 +131,245 @@ display:
...
+          operator: word
+          value: ''
           group: 1
-          exposed: false
+          exposed: true
           expose:
-            operator_id: ''
-            label: ''
+            operator_id: combine_op
+            label: Filter
```
[the remark this answers is not in the document]
Comment (idimopoulos) [reply web/profiles/joinup/config/install/views.view.members_overview.yml]: How about "Type something to filter the list"?

File: web/profiles/joinup/config/install/views.view.members_overview.yml [hunk 3]
```diff
@@ -122,83 +131,245 @@ display:
...
+          plugin_id: field
+      filters:
+        combine:
+          id: combine
+          table: views
+          field: combine
+          relationship: none
+          group_type: group
+          admin_label: ''
+          operator: word
```
[the remark this answers is not in the document]
Comment (idimopoulos) [reply web/profiles/joinup/config/install/views.view.members_overview.yml]: Indeed it seems a better choice! I will change this to all words.

File: web/profiles/joinup/config/install/views.view.members_overview.yml (hunk 1 above)
Comment (idimopoulos) [reply web/profiles/joinup/config/install/views.view.members_overview.yml]: I don't think it is would be a good idea to do so. We do not expose the emails in the interface and we do not allow users to see other user's emails. Allowing them to filter by email could unintentionally give users a tool to verify email addresses of other people. That is why I don't usually set up emails in searches. However, I will include this remark and the above one (the label of the search field) in the acceptance criteria of the ticket for the functional team to verify and provide their opinion.

**Pull request merged**